### PR TITLE
Added Search topic

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on https://help.stackrox.com/docs/search/

Preview: https://openshift-docs-git-search-gnelson.vercel.app/openshift-acs/master/operating/search-filter.html

Please review after https://github.com/openshift/openshift-docs/pull/33017 is merged.

### CODE REVIEW

File: modules/search-attributes.adoc [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+| Allowed value for a taint currently applied to a node.
+
+| Toleration Key
+| Key for a toleration applied to a deployment.
+
+| Toleration Value
+| Value for a toleration applied to a deployment.
+
+| Violation
+| A notification displayed in the *Violations* page when the conditions specified by a policy haven't been met.
```
Comment (bergerhoffer) [inline modules/search-attributes.adoc:263]: haven't -> have not (another instance)

File: modules/search-attributes.adoc [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+| Category
+| Policy categories include DevOps Best Practices, Security Best Practices, Privileges, Vulnerability Management, Multiple, and any custom policy categories that you create.
+//TODO: Add link to custom policy categories topic
+
+| Cert Expiration
+| Certificate expiration date.
+
+| Cluster
+| Name of a Kubernetes or OpenShift cluster.
```
Comment (bergerhoffer) [inline modules/search-attributes.adoc:40]: OpenShift -> {ocp} here and below

File: modules/search-attributes.adoc [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+| Drop Capabilities
+| Linux capabilities that have been dropped from the container.
+For example `CAP_SETUID` or `CAP_NET_RAW`.
+
+| Enforcement
+| Type of enforcement assigned to the deployment.
+For example, `None`, `Scale to Zero Replicas`, or `Add an Unsatisfiable Node Constraint`.
+
+| Environment Key
+| Key portion of a label key-value string that's metadata for further identifying and organizing the environment of a container.
```
Comment (bergerhoffer) [inline modules/search-attributes.adoc:84]: that’s -> that is? Other places, if so

File: modules/search-attributes.adoc [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+| Name of the process for a process indicator in a deployment.
+
+| Process Path
+| Path to the binary in the container for a process indicator in a deployment.
+
+| Process UID
+| Unix user ID for the process indicator in a deployment.
+
+| Read Only Root Filesystem
+| Use `true` to search for containers running with the root filesystem configured as read only.
```
Comment (bergerhoffer) [inline modules/search-attributes.adoc:215]: filesystem -> file system

File: modules/search-syntax.adoc [hunk 5]
```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * operating/search-filter.adoc
+:_module-type: CONCEPT
+[id="search-syntax_{context}"]
+= Search syntax
+
+A search query is made up of two parts:
```
Comment (bergerhoffer) [inline modules/search-syntax.adoc:8]: The list below isn't rendering because there isn't a blank line before it

File: modules/search-syntax.adoc [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+. An attribute that identifies the resource type you are after.
```
Comment (bergerhoffer) [inline modules/search-syntax.adoc:9]: * Probably should be an unnumbered list?
* "you are after" sounds a little casual. Could it be reworded, like "you want to search for" or something similar?

File: modules/search-syntax.adoc [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+. A search term that finds the matching resource.
+
+For example, to find all violations in the `visa-processor` deployment, the search query is, `Deployment:visa-processor`. In this search query, `Deployment` is the attribute and `visa-processor` is the search term.
```
Comment (bergerhoffer) [inline modules/search-syntax.adoc:12]: No comma after "is"

File: modules/use-local-page-filtering.adoc [hunk 8]
```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
```
Comment (bergerhoffer) [inline modules/use-local-page-filtering.adoc:1]: I don't see this file included in the assembly, should it be?

File: modules/common-search-queries.adoc [hunk 9]
```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * operating/search-filter.adoc
+:_module-type: CONCEPT
+[id="common-search-queries_{context}"]
+= Common search queries
+
+Here are some common search queries you can run with the StackRox Kubernetes Security Platform.
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply modules/common-search-queries.adoc:8]: 🤦🏼‍♂️

**Comment by gaurav-nelson** [comment]
@bergerhoffer 👍🏼 Let's get this merged.

**Review by bergerhoffer — COMMENTED** [review]
LGTM

**Pull request merged**